src/components/RelationshipPicker.tsx

```diff
@@ -49,4 +135,10 @@ const useStyles = makeStyles((theme) => ({
     flexDirection: 'column',
     alignItems: 'left',
   },
+  closeButton: {
```

Comment:
It looks like you're no longer using this?

Reply:
I missed that, thanks! I will remove the closeButton 👍